installer/lib/phx_new/generator.ex

```diff
@@ -23,28 +23,27 @@ defmodule Phx.New.Generator do
     root = Path.expand("../../templates", __DIR__)
 
     templates_ast =
-      for {name, mappings} <- Module.get_attribute(env.module, :templates) do
-        for {format, _proj_location, files} <- mappings,
-            format != :keep,
-            {source, _target} <- files,
-            source = to_string(source) do
-          path = Path.join(root, source)
-
-          if format in [:config, :prod_config, :eex] do
-            compiled = EEx.compile_file(path)
-
-            quote do
-              @external_resource unquote(path)
-              @file unquote(path)
-              def render(unquote(name), unquote(source), var!(assigns))
-                  when is_list(var!(assigns)),
-                  do: unquote(compiled)
-            end
-          else
-            quote do
-              @external_resource unquote(path)
-              def render(unquote(name), unquote(source), _assigns), do: unquote(File.read!(path))
-            end
+      for {name, mappings} <- Module.get_attribute(env.module, :templates),
+          {format, _proj_location, files} <- mappings,
+          format != :keep,
+          {source, _target} <- files,
+          source = to_string(source) do
```

Comment:
It's hard to see from the diff, but there was a nested `for` comprehension here. I merged them into one.